internal/pkg/deploy/task.go

```diff
@@ -22,3 +27,23 @@ type CreateTaskResourcesInput struct {
 
 	AdditionalTags map[string]string
 }
+
+// TaskStackInfo contains essential information about a Copilot task stack
+type TaskStackInfo struct {
+	StackName string
+	App       string
+	Env       string
+
+	RoleARN string
+}
+
+// TaskName returns the name of the one-off task. This is the same as the value of the
+// copilot-task tag.
+func (t TaskStackInfo) TaskName() string {
+	return strings.SplitN(t.StackName, "-", 2)[1]
+}
+
+// ECRRepoName returns the name of the ECR repo for the one-off task.
+func (t TaskStackInfo) ECRRepoName() string {
+	return fmt.Sprintf("copilot-%s", t.TaskName())
```

Comment:
in CLI pkg we have `fmtRepoName`. To make them consistent could you make task ECR repo format `copilot-%s` a constant and update the CLI pkg to reuse the format constant?